src/misc/pootle-migration.md

```diff
@@ -7,7 +7,7 @@ There are a few things you can do to make the transition run smoothly:
 2. Check your permissions on your profile to make sure they match what you’d expect.
 3. Flag issues with Pontoon to the l10n-drivers so that we can triage them and address them in a timely manner. To do this, please [file a bug here](https://goo.gl/tRQzAJ), or reach out to the l10n-drivers if you’re not yet comfortable with Bugzilla.
 
-If you’re a locale manager on Pootle, check the [community permissions](https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/users.html#managing-permissions) on Pontoon and make any necessary changes to mirror your community’s permission settings.
+If you’re a locale manager on Pootle, check the [community permissions](../tools/pontoon/users.md#managing-permissions) on Pontoon and make any necessary changes to mirror your community’s permission settings.
```

Comment:
I wanted to check these links, but I can't find the page?

There's more in `misc` that's not rendered? Probably both a question for @flodolo .

Reply:
That's by design. It's stuff that we don't need to be in GH pages, it's just there as an archive, and available just through the repository.

Reply:
Having said that, this is the only page not rendered (i.e. not in SUMMARY.md), unless I'm misunderstanding the question.